tableauserverclient/server/endpoint/schedules_endpoint.py

```diff
@@ -0,0 +1,60 @@
+from .endpoint import Endpoint
+from .exceptions import MissingRequiredFieldError
+from .. import RequestFactory, PaginationItem, ScheduleItem
+import logging
+import copy
+
+logger = logging.getLogger('tableau.endpoint.schedules')
+
+
+class Schedules(Endpoint):
+    def __init__(self, parent_srv):
+        super(Endpoint, self).__init__()
+        self.baseurl = "{0}/schedules"
+        self.parent_srv = parent_srv
+
+    def _construct_url(self):
```

Comment:
We changed the pattern here so that it was in sync with the root server class. We don't use _construct_url. We just made base_url a property and build the string when you call it. Let's be consistent. It was done after Chris wrote the original pass on this but we should update.